src/main/java/com/minecolonies/coremod/fixers/EventBasedIdFixer.java

```diff
@@ -0,0 +1,52 @@
+package com.minecolonies.coremod.fixers;
+
+import com.minecolonies.api.util.Log;
+import com.minecolonies.api.util.constant.Constants;
+import net.minecraft.block.Block;
+import net.minecraft.item.Item;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import net.minecraftforge.registries.IForgeRegistryEntry;
+import org.jetbrains.annotations.Nullable;
+
+@Mod.EventBusSubscriber(modid = Constants.MOD_ID)
+public class EventBasedIdFixer
+{
+    @SubscribeEvent
+    public static void onItemRegistryMissingMappings(final RegistryEvent.MissingMappings<Item> event)
+    {
+        Log.getLogger().warn("Remapping of minecolonies items started.");
```

Comment:
Wouldn't it try this for missing "non-minecolonies-blocks" as well?

Reply:
Sure
But that is why it logs what it updates.